atd-vze/src/Components/GridTable.js

```diff
@@ -526,15 +523,13 @@ const GridTable = ({
                     moveBack={moveBackPage}
                     pageNumber={page}
                     limit={limit}
-                    totalRecords={totalRecords}
-                    totalPages={totalPages}
                     handleRowClick={handleRowClick}
```

Comment:
One small bug that i noticed which I think already exists is that you can page ahead, update the the row count, and then end up on a page that doesn't exist like if you go from 10 rows per page to 100 rows per page for example. 

Resetting the pagination in this handler could be a way to handle this, but I think a spinoff is good since it is reproducible in the current prod editor.

Reply:
this is a duplicate of my above comment - not sure what happened 🙃